sdk/typescript/README.md

```diff
@@ -1,10 +1,10 @@
-# Dagger NodeJS SDK
+# Dagger TypeScript SDK
 
 A client package for running [Dagger](https://dagger.io/) pipelines.
 
-## What is the Dagger NodeJS SDK?
+## What is the Dagger TypeScript SDK?
 
-The Dagger NodeJS SDK contains everything you need to develop CI/CD pipelines in Typescript of Javascript, and run them on any OCI-compatible container runtime.
+The Dagger TypeScript SDK contains everything you need to develop CI/CD pipelines in TypeScript of Javascript, and run them on any OCI-compatible container runtime.
```

Comment:
```suggestion
The Dagger TypeScript SDK contains everything you need to develop CI/CD pipelines in TypeScript or Javascript, and run them on any OCI-compatible container runtime.
```

Reply:
Done in https://github.com/dagger/dagger/pull/6381/commits/5a635fc29b6b40c36fbe42fd315a12b29dd0a4be